pkgs/applications/office/envelope/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitHub
+, git
+, meson
+, ninja
+, pkgconfig
+, pantheon
+, python3
+, appstream-glib
+, desktop-file-utils
+, glib
+, gtk3
+, libgee
+, sqlite
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "envelope";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "cjfloss";
+    repo = pname;
+    rev = version;
+    sha256 = "111lq1gijcm7qwpac09q11ymwiw2x3m12a28ki52f28fb1amvffc";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pantheon.vala
+    pkgconfig
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    git
+    appstream-glib
```

Comment:
can you move this to `nativeBuildInputs` and also add `desktop-file-utils`?
* https://github.com/cjfloss/envelope/blob/master/data/meson.build#L33

On another note, I should probably package their vala linting tool 
* https://github.com/cjfloss/envelope/blob/master/data/meson.build#L51

Reply:
I saw the missing vala linting tool too. Could be great if we have it. Packaging it don't seem too difficult, so if I find the time I can maybe try to submit a PR too.

Reply:
You're in luck, I did this yesterday https://github.com/NixOS/nixpkgs/pull/69052/commits/c7cc5ab07bac39acf673d22c85ae21296b3ac2a2